discovery_engine/lib/src/api/events/engine_events.dart

```diff
@@ -132,6 +132,7 @@ class EngineEvent with _$EngineEvent {
   @Implements<SystemEngineEvent>()
   const factory EngineEvent.engineExceptionRaised(
     EngineExceptionReason reason,
+    String? message,
```

Comment:
what about adding stack trace? might be useful for the app

Reply:
I wasn't sure it can pass the isolate. I'll check and add it or change it to a string if possible.